packages/plugins/typescript-resolvers/tests/ts-resolvers.spec.ts

```diff
@@ -713,19 +799,32 @@ describe('TypeScript Resolvers Plugin', () => {
         const resolvers: Resolvers = {
           Subscription: {
             postAdded: {
-              // Additional event labels can be passed to asyncIterator creation
               subscribe: () => pubsub.asyncIterator([POST_ADDED]),
-            },
+              // Make sure we get and return a proper value
+              resolve: post => post
```

Comment:
The result of `subscribe()` is passed here, we know what it should pass to nested resolvers so it's all good here.